### PR TITLE
docs: persistent cache

### PR DESCRIPTION
**Pull request by jerrykingxyz**
## Summary

Add document for `persistent_cache` feature.

## Checklist

- [ ] Tests updated (or not required).
- [x] Documentation updated (or not required).

### CODE REVIEW

**Review by chenjiahan — COMMENTED** [review]
This PR contains some non-doc changes, can you split them to another PR?

File: website/docs/en/config/experiments.mdx [hunk 1]
```diff
@@ -334,3 +334,150 @@ module.exports = {
...
+        directory?: string;
+      };
+    };
+```
+
+Control experimental caching behavior. This will only work if [global cache](/config/cache) is set to `true`.
+
+### Disable cache
+
+Configuring `experiment.cache` to `false` to disable cache, which is no different from configuring the [global cache](/config/cache) to `false`.
```
Comment (chenjiahan) [inline website/docs/en/config/experiments.mdx:372]: The Info in `/config/cache` can be updated now:

image

File: website/docs/en/config/experiments.mdx [hunk 2]
```diff
@@ -334,3 +334,150 @@ module.exports = {
   },
 };
 ```
+
+## experiments.cache
```
Comment (chenjiahan) [inline website/docs/en/config/experiments.mdx:338]: We can follow the alphabetical order and put the `cache` option before `experiments.css`

File: website/docs/en/config/experiments.mdx [hunk 3]
```diff
@@ -334,3 +334,150 @@ module.exports = {
...
+
+##### snapshot.unmanagedPaths
+
+- **Type:** `(RegExp | string)[]`
+
+- **Default:** `[]`
+
+Specifies an array of paths in `snapshot.managedPaths` that are not managed by the package manager
+
+#### cache.storage
```
Comment (chenjiahan) [inline website/docs/en/config/experiments.mdx:467]: Missing `Type` and `Default` here

File: website/docs/en/config/experiments.mdx (hunk 2 above)
Comment (jerrykingxyz) [reply website/docs/en/config/experiments.mdx:338]: Currently they are arranged in the order of supported versions, we can modify them in another PR

Comment (chenjiahan) [reply website/docs/en/config/experiments.mdx:338]: ok

**Review by chenjiahan — APPROVED** [review]

**Pull request merged**